File: Palindrome_Number.java
class Solution {
    public boolean isPalindrome(int x) {
        if ( x < 0 ){
            return false;
        }
        String s = Integer.toString(x);
        StringBuilder sb = new StringBuilder();
        sb.append(s);
        sb = sb.reverse();
        String rs = sb.toString();
        if ( s.equals(rs) ){
            return true;
        }
        else {
            return false;
        }
    }
}

File: Coding Problems & My Solutions/Palindrome Number/Palindrome_Number.kt
class Solution {
    fun isPalindrome(x: Int): Boolean {
        if (x < 0) { return false }
        val str = x.toString()
        var a = 0
        var b = str.length - 1
        while (a <= b) {
            if (str[a] != str[b]) {
                return false
            }
            a += 1
            b -= 1
        }
        return true
    }
}